### PR TITLE
Latest swagger updates

### LINKED ISSUE #1944: Add State to Semantic Actions in Schema
The State field needs to be added to the Schema for Semantic Actions

This tracks Issue:
https://github.com/microsoft/botframework-sdk/issues/5403

### PR DESCRIPTION
**Pull request by EricDahlvang**
Fixes #1944

### CODE REVIEW

File: libraries/Microsoft.Bot.Schema/Activity.cs [hunk 1]
```diff
@@ -131,7 +131,12 @@ public Activity()
...
...n> reactionsRemoved = default(IList<MessageReaction>), string topicName = default(string), bool? historyDisclosed = default(bool?), string locale = default(string), string text = default(string), string speak = default(string), string inputHint = default(string), string summary = default(string), SuggestedActions suggestedActions = default(SuggestedActions), IList<Attachment> attachments = default(IList<Attachment>), IList<Entity> entities = default(IList<Entity>), object channelData = default(object), string action = default(string), string replyToId = default(string), string label = default(string), string valueType = default(string), object value = default(object), string name = default(string), ConversationReference relatesTo = default(ConversationReference), string code = default(string), System.DateTimeOffset? expiration = default(System.DateTimeOffset?), string importance = default(string), string deliveryMode = default(string), IList<string> listenFor = default(IList<string>), IList<TextHighlight> textHighlights = default(IList<TextHighlight>), SemanticAction semanticAction = default(SemanticAction), string localTimezone = default(string), string callerId = default(string))
```
Comment (cleemullins) [inline libraries/Microsoft.Bot.Schema/Activity.cs]: We can't do this. Adding a constructor parameter is a breaking change. 

What we do about this, I'm not sure. :(

File: libraries/Microsoft.Bot.Schema/SemanticAction.cs [hunk 2]
```diff
@@ -33,10 +33,12 @@ public SemanticAction()
         /// </summary>
         /// <param name="id">ID of this action</param>
         /// <param name="entities">Entities associated with this action</param>
-        public SemanticAction(string id = default(string), IDictionary<string, Entity> entities = default(IDictionary<string, Entity>))
+        /// <param name="state">State of this action. Allowed values: `start`, `continue`, `done`</param>
+        public SemanticAction(string id = default(string), IDictionary<string, Entity> entities = default(IDictionary<string, Entity>), string state = default(string))
```
Comment (cleemullins) [inline libraries/Microsoft.Bot.Schema/SemanticAction.cs]: Also can't do this. 

Have to have an additional ctor where "state" is required. (aka: overloads are ok)

File: libraries/Microsoft.Bot.Schema/Activity.cs (hunk 1 above)
Comment (cleemullins) [reply libraries/Microsoft.Bot.Schema/Activity.cs]: Have to make this a new ctor with the additional parameter required, then have 1 call the other.

**Comment by fuselabs** [comment]
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.Luis.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.QnA.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.ApplicationInsights.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Azure.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Dialogs.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.TemplateManager.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Configuration.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Connector.dll**
:x: 2 Binary Compatibility issues for **Microsoft.Bot.Schema**
<details>

```
MembersMustExist : Member 'Microsoft.Bot.Schema.Activity..ctor(System.String, System.String, System.Nullable<System.DateTimeOffset>, System.Nullable<System.DateTimeOffset>, System.String, System.String, Microsoft.Bot.Schema.ChannelAccount, Microsoft.Bot.Schema.ConversationAccount, Microsoft.Bot.Schema.ChannelAccount, System.String, System.String, System.Collections.Generic.IList<Microsoft.Bot.Schema.ChannelAccount>, System.Collections.Generic.IList<Microsoft.Bot.Schema.ChannelAccount>, System.Collections.Generic.IList<Microsoft.Bot.Schema.MessageReaction>, System.Collections.Generic.IList<Microsoft.Bot.Schema.MessageReaction>, System.String, System.Nullable<System.Boolean>, System.String, System.String, System.String, System.String, System.String, Microsoft.Bot.Schema.SuggestedActions, System.Collections.Generic.IList<Microsoft.Bot.Schema.Attachment>, System.Collections.Generic.IList<Microsoft.Bot.Schema.Entity>, System.Object, System.String, System.String, System.String, System.String, System.Object, System.String, Microsoft.Bot.Schema.ConversationReference, System.String, System.Nullable<System.DateTimeOffset>, System.String, System.String, System.Collections.Generic.IList<System.String>, System.Collections.Generic.IList<Microsoft.Bot.Schema.TextHighlight>, Microsoft.Bot.Schema.SemanticAction, System.String)' does not exist in the implementation but it does exist in the contract.
MembersMustExist : Member 'Microsoft.Bot.Schema.SemanticAction..ctor(System.String, System.Collections.Generic.IDictionary<System.String, Microsoft.Bot.Schema.Entity>)' does not exist in the implementation but it does exist in the contract.
```

</details>

File: libraries/Microsoft.Bot.Schema/SemanticAction.cs (hunk 2 above)
Comment (EricDahlvang) [reply libraries/Microsoft.Bot.Schema/SemanticAction.cs]: removed the state parameter

File: libraries/Microsoft.Bot.Schema/Activity.cs (hunk 1 above)
Comment (EricDahlvang) [reply libraries/Microsoft.Bot.Schema/Activity.cs]: Removed the callerId parameter. 

This constructor is unwieldy.

**Comment by fuselabs** [comment]
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.Luis.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.QnA.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.ApplicationInsights.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Azure.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Dialogs.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.TemplateManager.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Configuration.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Connector.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Schema.dll**

**Pull request merged**